server/src/main/java/org/opensearch/env/NodeEnvironment.java

```diff
@@ -123,14 +133,20 @@ public static class NodePath {
         public final Path indicesPath;
         /** Cached FileStore from path */
         public final FileStore fileStore;
-
+        public final Path cachePath;
```

Comment:
Can we be consistent with the naming? It looks like there is a mix of "cache", "fileCache", and "remoteStoreFileCache" throughout. For now, I suggest normalizing this to "fileCache" since that is the class name as well.

Reply:
I named the variable as per context, naming it to `remoteStoreFileCache` where we have the required information available. I'll rename it to `fileCache` to simplify since there is no other file cache usage as of now.